.github/workflows/release.yaml

```diff
@@ -0,0 +1,32 @@
+name: Upload acme.js to release page
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    name: Build release artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Build dist/acme.js
+        run: make build
+
+      - name: Upload acme.js to release
+        uses: actions/upload-release-asset@v1.0.2
```

Comment:
From their [readme](https://github.com/tanyagray/action-upload-release-asset/blob/main/README.md): 

> Please note: This repository is currently unmaintained by a team of developers at GitHub. The repository is here and you can use it as an example, or in Actions. However please be aware that we are not going to be updating issues or pull requests on this repository.

Might want to find a different action, or use the [`gh release upload`](https://cli.github.com/manual/gh_release_upload) CLI - the runner should have it installed already. I think that'd look like (⚠️ untested ⚠️):

```yaml
      - name: Upload acme.js to release
        run: gh release upload "${{ github.event.release.tag_name }}" ./dist/acme.js
```

Reply:
Thanks for both of these, @ryepup ! I've simplified the file quite a bit and tested in a temp repo.
https://github.com/zsteinkamp/njs-acme-test/releases/tag/v1.0.2
https://github.com/zsteinkamp/njs-acme-test/blob/main/.github/workflows/release.yaml